sdk/storage/CONTRIBUTING.md

```diff
@@ -18,28 +18,35 @@ The Azure Storage development team uses Visual Studio Code. However, any preferr
+  | ACCOUNT_KEY               | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the key.                                                                                                                                                                                                                                                                                              | Required                                                                         |
+  | ACCOUNT_SAS               | Can be obtained from the Azure portal. Go to the `Shared access signature` tab under the `Settings` section of your storage account in the portal. Generate SAS by enabling permissions, setting exiry dates, etc. You can also generate a valid account SAS from tools like Azure Storage Explorer. A SAS starts with "?". And if you are using Windows CMD, you may need quotes to escape special characters like this - `set "ACCOUNT_SAS=<YOUR_SAS>"` | Required to run the browser tests                                                |
+  | ACCOUNT_TOKEN             | Can you provide some description refferring the proper way to get this, @jiacfan?                                                                                                                                                                                                                                                                                                                                                                         | Not required (Some of the tests will be skipped if this variable is not defined) |
+  | STORAGE_CONNECTION_STRING | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the connection string.                                                                                                                                                                                                                                                                                | Required                                                                         |
+  |                           |                                                                                                                                                                                                                                                                                                                                                                                                                                                           |                                                                                  |
 
 The environment variable **TEST_MODE** controls how tests are running.
```

Comment:
it feels weird to introduce the notion of `TEST_MODE` above and then have to scroll past the environment variable table to get to the description of what the test modes actually are/do. I would consider bumping down the environment variable section or giving a brief sentence or two about the modes earlier.

Reply:
Even something simple like: `"live" is used to test against the service directly without recording, "playback" is used to test against earlier recordings, and "record" is used to test against the service while recording responses.` before we get into the algorithm of when each mode is used and the implementation details

Reply:
Thanks for this suggestion, updated accordingly(and appropriately :P) - 7f189c1